web/packages/teleterm/src/services/appUpdater/appUpdater.ts

```diff
@@ -100,6 +100,7 @@ export class AppUpdater {
     this.nativeUpdater.logger = this.logger;
     this.nativeUpdater.allowDowngrade = true;
     this.nativeUpdater.autoDownload = false;
+    this.versionEnvVar = versionEnvVar;
```

Comment:
This one isn't actually needed, notice how we don't have an equivalent lines for `storage` or `getClusterVersions`. Grzegorz taught me this a long time ago.

https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd

Reply:
oh wow, interesting. i thought even what i had was cool defining in the constructor cuz i didnt even know that. i was doing the oldschool way of defining the type and setting in the constructor. double TIL